src/Compilers/CSharp/Portable/Symbols/TypeWithState.cs

```diff
@@ -39,7 +42,7 @@ public static TypeWithState Create(TypeWithAnnotations typeWithAnnotations, Flow
             {
                 if ((annotations & FlowAnalysisAnnotations.MaybeNull) == FlowAnalysisAnnotations.MaybeNull)
                 {
-                    state = NullableFlowState.MaybeNull;
+                    state = NullableFlowState.MaybeDefault;
```

Comment:
>MaybeDefault [](start = 46, length = 12)

I think this state is only for unconstrained types.

Reply:
`Create()` use `MaybeNull` instead if necessary.

---
In reply to: [347695616](https://github.com/dotnet/roslyn/pull/39778#discussion_r347695616) [](ancestors = 347695616)